app/src/js/components/BatchAsyncCommands/BatchAsyncCommands.js

```diff
@@ -263,7 +265,11 @@ export class BatchCommand extends React.Component {
             }
             {(!inflight && !status) && (!modalOptions || !modalOptions.children) && (
               <>
-                <div>{confirmText}</div>
+                <div>{confirmTextArray.map((confirmText, index) => (
+                  <React.Fragment key={index}>
+                    {confirmText}
+                  </React.Fragment>
+                ))}</div>
```

Comment:
Made this change because there was a console warning about a lack of unique keys for elements in a list